pyproject.toml

```diff
@@ -122,7 +122,7 @@ sentry-sdk = ">=0.17.0,<1.3.0"
 aio-pika = "^6.7.1"
 pyTelegramBotAPI = "^3.7.3"
 tarsafe = "^0.0.3"
-google-auth = "^2.3.3"
+google-auth = "<2"
```

Comment:
Do we need any follow-ups to change the requirement back to a `>=2` release?

Reply:
@tttthomasssss I don't know. 

@TyDunn fyi, we did this change only because of #10257 and we wanted to upgrade to TF 2.6.2 to fix some security vulnerabilities. Enable squad may need to change this to `>=2` when they attempt to upgrade to TF >2.7 later.